### ISSUE TITLE
Activity icon in DRD blocks interaction

### ISSUE THREAD

**Issue by andreasgeier** [camunda/camunda-modeler#2804]
__Describe the Bug__

In the split screen, the symbols on the DMN elements are blocking the click event to switch the view.

__Steps to Reproduce__

1. create a DRD with at least two elements
2. open a DMN element in split view
3. try to switch the view by clicking on the icons in the elements

https://user-images.githubusercontent.com/938244/156809977-85f75b15-be38-4b1a-aa0a-2cb78abf8b0b.mov

__Expected Behavior__

Basically, it shouldn't matter where the user moves the mouse pointer over the element in the DRD or clicks within the element. The symbol area must not block this interaction.

__Environment__

 * Version: 5.0.0-nightly.20220303
 * Operating System: Mac OS 10.15.7 
 * Plugins: <no plugins>
 * Execution Platform: DMN - Camunda Platform or Cloud

**Comment by andreasgeier** [comment]
Reported by @falko via [Slack](https://camunda.slack.com/archives/CKGH9LR40/p1646407469350579)

**Comment by smbea** [comment]
I can reproduce this. Will move to backlog
